include/ros_control_boilerplate/generic_hw_interface.h

```diff
@@ -80,9 +80,23 @@ class GenericHWInterface : public hardware_interface::RobotHW
   /** \brief Read the state from the robot hardware. */
   virtual void read(ros::Duration &elapsed_time) = 0;
 
+  /** \brief Delegate RobotHW::read() calls to read() */
+  virtual void read(const ros::Time& /*time*/, const ros::Duration& period)
```

Comment:
Can you please fully doxygenize the comments to explain the arguments better? It's not clear to me what the difference is between elapsed_time (L81), time, and period. Here's my best guess:

```
 /** \brief Delegate RobotHW::read() calls to read() 
  *  \param time Current ROS time
  *  \param period Nominal control loop period
  */
  virtual void read(const ros::Time& /*time*/, const ros::Duration& period)
```

Reply:
Sure, i just wanted to make it consistent with the rest of the functions.